### PR TITLE
Enhance ADMIN.md

### PR DESCRIPTION
**Pull request by chri2**
changes about artifact-expiry triggered a rebuild

https://gitlab.com/famedly/conduit/-/commit/13ae7a7e4633370941b73c3a65f5969f42756ec1

### CODE REVIEW

**Comment by chri2** [comment]
!testme

**Comment by tituspijean** [comment]
Superseded by #31 and #34, kept for the documentation update

File: doc/ADMIN.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+To be able to take advantage of audio and video call functionalities, a coturn server is often required. It is possible to [install a coturn server in YunoHost (https://github.com/YunoHost-Apps/coturn-ynh/blob/master/README-en.md).
+It is then necessary to fill in the information provided by the coturn server in the file 'conduit.toml' such as:
+
+```
+turn_uris = ["turns:your.turn.url:5349?transport=udp", "turns:your.turn.url:5349?transport=tcp"]
+turn_username = "<YOUR_USERNAME>"
+turn_password = "<YOUR_PASSWORD>"
+```
+If your coturn (not Yunohost's one) don't use TLS, you might need to change a little bit like :
```
Comment (ericgaspar) [inline doc/ADMIN.md:17]:
```suggestion
If your Coturn (not YunoHost's one) don't use TLS, you might need to change a little bit like:
```

File: doc/ADMIN.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+### Login to conduit
+
+[Integration into the yunohost user database is not ready, yet.](https://github.com/YunoHost-Apps/conduit_ynh/issues/12) You need to **enable registration** in the apps configuration tab and then use a client to register an account. Afterwards you can disable registration again if you do not want allow anybody to register.
+
+Using the newly created account you should be able to login.
+
+###  Coturn configuration
+
+To be able to take advantage of audio and video call functionalities, a coturn server is often required. It is possible to [install a coturn server in YunoHost (https://github.com/YunoHost-Apps/coturn-ynh/blob/master/README-en.md).
```
Comment (ericgaspar) [inline doc/ADMIN.md:9]:
```suggestion
To be able to take advantage of audio and video call functionalities, a Coturn server is often required. It is possible to [install a Coturn server in YunoHost (https://github.com/YunoHost-Apps/coturn-ynh/blob/master/README-en.md).
```

File: doc/ADMIN.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### Login to conduit
```
Comment (ericgaspar) [inline doc/ADMIN.md:1]:
```suggestion
### Login to Conduit
```

File: doc/ADMIN.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+[Integration into the yunohost user database is not ready, yet.](https://github.com/YunoHost-Apps/conduit_ynh/issues/12) You need to **enable registration** in the apps configuration tab and then use a client to register an account. Afterwards you can disable registration again if you do not want allow anybody to register.
+
+Using the newly created account you should be able to login.
+
+###  Coturn configuration
+
+To be able to take advantage of audio and video call functionalities, a coturn server is often required. It is possible to [install a coturn server in YunoHost (https://github.com/YunoHost-Apps/coturn-ynh/blob/master/README-en.md).
+It is then necessary to fill in the information provided by the coturn server in the file 'conduit.toml' such as:
```
Comment (ericgaspar) [inline doc/ADMIN.md:10]:
```suggestion
It is then necessary to fill in the information provided by the Coturn server in the file 'conduit.toml' such as:
```

File: doc/ADMIN.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+### Login to conduit
+
+[Integration into the yunohost user database is not ready, yet.](https://github.com/YunoHost-Apps/conduit_ynh/issues/12) You need to **enable registration** in the apps configuration tab and then use a client to register an account. Afterwards you can disable registration again if you do not want allow anybody to register.
```
Comment (ericgaspar) [inline doc/ADMIN.md:3]:
```suggestion
[Integration into the YunoHost user database is not ready, yet.](https://github.com/YunoHost-Apps/conduit_ynh/issues/12) You need to **enable registration** in the apps configuration tab and then use a client to register an account. Afterwards you can disable registration again if you do not want allow anybody to register.
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**